lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
+
+            var event = db.events.get(req.params.id);
+
+            var eventTitle = event.get("title");
+            var history = event.get("history");
+            var participantCount = req.params.participantCount; 
+
+            var allUserIds = [];
+            allUserIds.push(null);
+
+            _.each(history.event, function(elapsedTime, userId) {
+                
+                //Get the user object for a specific ID 
+                var user = db.users.get(userId);
+
+                // Find all userIDs for users who shared sessions with me at this event.
+                var userIds = event.getUserIdsSharingSessionsWith(userId);
+                
+                // get user objects for the IDs.
+                var users = _.map(userIds, function(userId) { 
+                    return db.users.get(userId); 
+                });
+
+                allUserIds.push({ user: user, users: users });
+
+            });
+
+            var allUserIdsCount = Object.keys(allUserIds).length - 1;
+
+            res.render("email_preview_template.ejs", {
+                eventTitle: eventTitle,
+                participantCount: participantCount,
+                allUserIds: allUserIds,
+                allUserIdsCount: allUserIdsCount,
+                event: event
+            });
+        });
+
+        app.post("/followup/event/:id/sent/", function(req, res) {
+
+            var event = db.events.get(req.params.id);
+            var eventTitle = event.get("title");
+            var history = event.get("history");
+
+            var allUserIds = [];
+
+            _.each(history.event, function(elapsedTime, userId) {
+                
+                //Get the user object for a specific ID 
+                var user = db.users.get(userId);
+
+                // Find all userIDs for users who shared sessions with me at this event.
+                var userIds = event.getUserIdsSharingSessionsWith(userId);
+                
+                // get user objects for the IDs.
+                var users = _.map(userIds, function(userId) { 
+                    return db.users.get(userId); 
+                });
+
+                allUserIds.push({ user: user, users: users });
+
+            }); 
+
+            var allUserIdsCount = Object.keys(allUserIds).length;
+            var mandrill_client = new mandrill.Mandrill('PLDkuaJHZNqb2iuhLSGzYQ');
```

Comment:
API key needs to not be hard-coded, and not checked into git.  If this API key is live (e.g. can be used to send mail), invalidate it IMMEDIATELY.